src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -139,31 +139,18 @@
     BoxClient boxClient;
 
     private AvatarController avatarController;
+    private AboutView aboutView;
+    private AccountingView accountingView;
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         resourceBundle = resources;
 
         navi.getChildren().clear();
-        AccountingView accountingView = new AccountingView();
-        actionlogView = new ActionlogView();
-        accountingNav = createNavItem(resourceBundle.getString("layoutIdentity"),
-            new Image(getClass().getResourceAsStream("/img/account_white.png")),
-            accountingView);
-        navi.getChildren().add(accountingNav);
-        browseNav = createNavItem(resourceBundle.getString("layoutBrowse"),
-            new Image(getClass().getResourceAsStream("/img/folder_white.png")),
-            new RemoteFSView());
-        contactsNav = createNavItem(resourceBundle.getString("layoutContacts"),
-            new Image(getClass().getResourceAsStream("/img/account_multiple_white.png")),
-            new ContactView());
-        syncNav = createNavItem(resourceBundle.getString("layoutSync"),
-            new Image(getClass().getResourceAsStream("/img/sync_white.png")),
-            new SyncView());
-        aboutNav = createNavItem(resourceBundle.getString("layoutAbout"),
-            new Image(getClass().getResourceAsStream("/img/information_white.png")),
-            new AboutView());
 
+        createNavItems();
```

Comment:
besides that this never was very beautiful you previously had to touch one place to add a new nav item. Now the creation and adding of a new nav item is distributed over two methods. I would prefer to move the `add`s to where the items are created to have it in one place again. You could also let `createNavItem` add the item to the navi given all items are added.